deployment/ccip/test_usdc_helpers.go

```diff
@@ -0,0 +1,222 @@
+		return nil, nil, err
+	}
+
+	return srcToken, dstToken, nil
+}
+
+func UpdateFeeQuoterForUSDC(
+	chain deployment.Chain,
+	state CCIPChainState,
+	dstChain uint64,
+	usdcToken *burn_mint_erc677.BurnMintERC677,
+) error {
+	config := []fee_quoter.FeeQuoterTokenTransferFeeConfigArgs{
+		{
+			DestChainSelector: dstChain,
+			TokenTransferFeeConfigs: []fee_quoter.FeeQuoterTokenTransferFeeConfigSingleTokenArgs{
+				{
+					usdcToken.Address(),
+					fee_quoter.FeeQuoterTokenTransferFeeConfig{
+						MinFeeUSDCents:    50,
+						MaxFeeUSDCents:    50_000,
+						DeciBps:           0,
+						DestGasOverhead:   180_000,
+						DestBytesOverhead: 640,
+						IsEnabled:         true,
+					},
+				},
+			},
+		},
+	}
+
+	tx, err := state.FeeQuoter.ApplyTokenTransferFeeConfigUpdates(
+		chain.DeployerKey,
+		config,
+		[]fee_quoter.FeeQuoterTokenTransferFeeConfigRemoveArgs{},
+	)
+	if err != nil {
+		return err
+	}
+
+	_, err = chain.Confirm(tx)
+	return err
+}
+
+func DeployUSDC(
+	lggr logger.Logger,
+	chain deployment.Chain,
+	addresses deployment.AddressBook,
+) (
+	*burn_mint_erc677.BurnMintERC677,
+	*usdc_token_pool.USDCTokenPool,
+	*mock_usdc_token_messenger.MockE2EUSDCTokenMessenger,
+	*mock_usdc_token_transmitter.MockE2EUSDCTransmitter,
+	error,
+) {
+	token, err := deployContract(lggr, chain, addresses,
+		func(chain deployment.Chain) ContractDeploy[*burn_mint_erc677.BurnMintERC677] {
+			tokenAddress, tx, tokenContract, err2 := burn_mint_erc677.DeployBurnMintERC677(
+				chain.DeployerKey,
+				chain.Client,
+				"USDC Token",
+				"USDC",
+				uint8(18),
+				big.NewInt(0).Mul(big.NewInt(1e9), big.NewInt(1e18)),
+			)
+			return ContractDeploy[*burn_mint_erc677.BurnMintERC677]{
+				tokenAddress, tokenContract, tx, deployment.NewTypeAndVersion(USDCToken, deployment.Version1_0_0), err2,
+			}
+		})
+	if err != nil {
+		return nil, nil, nil, nil, err
+	}
+
+	tx, err := token.Contract.GrantMintRole(chain.DeployerKey, chain.DeployerKey.From)
+	if err != nil {
+		return nil, nil, nil, nil, err
+	}
+	_, err = chain.Confirm(tx)
+	if err != nil {
+		return nil, nil, nil, nil, err
+	}
+
+	transmitter, err := deployContract(lggr, chain, addresses,
+		func(chain deployment.Chain) ContractDeploy[*mock_usdc_token_transmitter.MockE2EUSDCTransmitter] {
+			transmitterAddress, tx, transmitterContract, err2 := mock_usdc_token_transmitter.DeployMockE2EUSDCTransmitter(
+				chain.DeployerKey,
+				chain.Client,
+				0,
+				reader.AllAvailableDomains()[chain.Selector],
+				token.Address,
+			)
+			return ContractDeploy[*mock_usdc_token_transmitter.MockE2EUSDCTransmitter]{
+				transmitterAddress, transmitterContract, tx, deployment.NewTypeAndVersion(USDCMockTransmitter, deployment.Version1_0_0), err2,
+			}
+		})
+	if err != nil {
+		lggr.Errorw("Failed to deploy mock USDC transmitter", "err", err)
+		return nil, nil, nil, nil, err
+	}
+
```

Comment:
Doesn't seem like these tx's are `chain.Confirm`d, do they need to be?

Reply:
DeployContract doesn't need to be confirmed, at least this is how it works for other contract deployment